metadata-ingestion/src/datahub/ingestion/source/looker/lookml_source.py

```diff
@@ -1265,13 +1321,31 @@ def _get_connection_def_based_on_connection_string(
     def _get_upstream_lineage(
         self, looker_view: LookerView
     ) -> Optional[UpstreamLineage]:
-        upstreams = []
+        # Merge dataset upstreams with sql table upstreams.
+        upstream_dataset_urns = []
+        for upstream_explore in looker_view.upstream_explores:
+            # We're creating a "LookerExplore" just to use the urn generator.
+            upstream_dataset_urn = LookerExplore(
+                name=upstream_explore, model_name=looker_view.id.model_name
+            ).get_explore_urn(self.source_config)
+            upstream_dataset_urns.append(upstream_dataset_urn)
```

Comment:
Nice !
So we will now get table level and column level lineage for native derived tables, i.e. below edge ?
 looker explore(upstream) -> looker view (derived)

Reply:
yep